demos/_GETTINGSTARTED.md

```diff
@@ -1,94 +1,44 @@
 ## Getting Started with Britecharts
 
-In this document, we show you the 3 main use cases you as developer will find when working with Britecharts:
-
+Britecharts has been created to help users consume and create d3 charts. It is based on the leverage of the reusable API. This produces chart objects that after initialized can be configured with a set of attributes, before applying the chart to a container with the data to plot.
```

Comment:
Britecharts has been created to help users consume and create d3 charts. `It leverages a reusable API, which produces chart objects that (after initialization) can be configured and then applied to a container with a data set to plot.`